templates/page.html

```diff
@@ -324,4 +324,88 @@ <h6>27 December 2019</h6>
     </div>
   </div>
 </section>
+<section class="roadmap-section section-content-white-bg" id="roadmap">
+  <div class="container">
+    <span class="section-title-white-bg">Roadmap</span>
+    <div class="row">
+      <div class="col-md-4">
+        <h3>Current</h3>
+        <p>
+          What we work on now
+        </p>
+        <div class="card">
+          <div class="card-body">
+            <h5 class="card-title">
+              <a href="https://github.com/orgs/reanahub/projects/33" class="card-link">
+                REANA v0.7.0
+              </a>
+            </h5>
+            <p class="card-text">
+              Prepare new Helm-based REANA 0.7.0 release.
+            </p>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card-body">
+            <h5 class="card-title">
+              <a href="https://github.com/orgs/reanahub/projects/34" class="card-link">
+                User quotas
+              </a>
+            </h5>
+            <p class="card-text">
+              Collect CPU and HDD consumption information for users running notebooks and workflows.
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="col-md-4">
+        <h3>Near-term</h3>
+        <p>
+          What we plan to work on next
+        </p>
+        <div class="card">
+          <div class="card-body">
+            <h5 class="card-title">Live logs</h5>
+            <p class="card-text">
+              Introduce live job log streaming for CLI and Web UI.
+            </p>
+          </div>
+        </div>
+        <div class="card">
+          <div class="card-body">
+            <h5 class="card-title">LHC community</h5>
+            <p class="card-text">
+              Dataset file abstraction
```

Comment:
We need to turn these into text, e.g. short phrases as elsewhere.  Example:

- Introduce abstract dataset concept to handle a set of related files.
- Improve live debugging of Yadage workflows.